test/journeys/specs/smoke/widget-recents/index.js

```diff
@@ -308,6 +308,18 @@ describe('Smoke Tests - Recents Widget', () => {
         assert.isNotEmpty(event.email, 'profile:clicked data does not contain email');
         assert.isNotEmpty(event.orgId, 'profile:clicked data does not contain orgId');
       });
+
+      it('user_signout:clicked', () => {
```

Comment:
I think your test failure is because you didn't init the widget with the profile popover enabled. Check line 64

Reply:
I'm trying.

Reply:
it worked.